cql3/ut_name.cc

```diff
@@ -40,7 +40,7 @@ sstring ut_name::get_string_type_name() const
 }
 
 sstring ut_name::to_string() const {
-    return (has_keyspace() ? (_ks_name.value() + ".") : "") + _ut_name->to_string();
+    return (has_keyspace() ? (_ks_name.value() + ".") : "") + _ut_name->to_cql_string();
```

Comment:
Should you perhaps consider renaming `ut_name::to_string()` to `ut_name::to_cql_string()`? Otherwise it's not obvious that this is what it does.

Reply:
I just added this, but to solve the problem completely, do we also change the `cql3_type::raw::to_string()` to `cql3_type::raw::to_cql_string()`? Or maybe there's no need for it here because we already have the `cql` in `cql3_type`?

Reply:
The point of the `to_cql_string` name makes a lot of sense. On the other hand, do we also have some specific interpretation for the `to_string` methods? 
Based on `to_cql_string` alone, it does make sense to change the name in `ut_name` and would be also more fitting in `cql3_type::raw` but there it would require more adjustments, and it's probably not worth pursuing in this patch